pkgs/applications/office/sleek/snap/gui/sleek.desktop.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/snap/gui/sleek.desktop b/snap/gui/sleek.desktop
+index ce43c15..decea7d 100755
+--- a/snap/gui/sleek.desktop
++++ b/snap/gui/sleek.desktop
+@@ -1,9 +1,9 @@
+ [Desktop Entry]
+ Name=sleek
+-Exec=sleek %U
++Exec=sleek
```

Comment:
Why not forward args?

Reply:
You're right. I didn't understand what it does yesterday and made a stupid decision.